extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -514,6 +520,16 @@ export class InstrumentationService {
     this.win_.setTimeout(this.win_.clearInterval.bind(this.win_, intervalId),
         maxTimerLength * 1000);
   }
+
+  isTriggerAllowed_(triggerType, element) {
+    if (!isInIframe(element) ||
+      triggerType == AnalyticsEventType.VISIBLE ||
+      triggerType == AnalyticsEventType.TIMER ||
+      triggerType == AnalyticsEventType.HIDDEN ||
+      triggerType == AnalyticsEventType.CLICK) {
+      return true;
```

Comment:
`return false` is missing.

That being said, I'd rather follow a "case -> rule strategy" here. E.g.

```
  if (isInIFrame()) {
    return IFRAME_WHITELIST[triggerType];
  }
  // by default everything is allowed
  return true;
```